### DISCUSSION TITLE
holiday calculation

### DISCUSSION THREAD

**Discussion by wobouser** [kimai/Paid-plugins#191] [Questions & Answers] [answered]
hi,

in the profile of an employee i have the possibility to enter the vacation per year and the start date of his company entry.
see picture below.

<img width="494" alt="image" src="https://github.com/user-attachments/assets/54430407-10aa-433b-b80b-b14e86d7e50b">

so he still has 10 days for the rest of the year

but in Employment contract -> Absence he still has 30 days

<img width="293" alt="image" src="https://github.com/user-attachments/assets/a464755a-068e-496c-8d0f-61c45520af8a">

according to this, i have to calculate the vacation for the employee myself every year ?

is the holiday entitlement of 30 days automatically carried over into the new year or do i have to enter it myself at the beginning of the year?

**Comment by kevinpapst** [comment, marked as answer]
> according to this, i have to calculate the vacation for the employee myself every year ?

Yes. Kimai does not calculate holidays. The logic would be country specific and that is not (yet) supported.

> is the holiday entitlement of 30 days automatically carried over into the new year or do i have to enter it myself at the beginning of the year?

You have to manually carry over holidays left-overs from previous years.

**Comment by wobouser** [comment]
ok, thx
can be closed
